test/tests/transfers/general_field/nearest_node/mesh_division/main_match_division.i

```diff
@@ -0,0 +1,168 @@
+# Base input for testing transfers with mesh divisions restrictions with a mapping from
+# source mesh divisions to target mesh divisions. It has the following complexities:
+# - multiple sub-applications
+# - transfers both from and to the subapps
+# - both nodal and elemental variables
+# Tests derived from this input may add complexities through command line arguments
+
+# Explaining results on the main app:
+# Each value on the main app comes from a region with the same division index on a subapp
+# Because the subapp is not very discretized in Y, some source mesh divisions index
+# are not represented in the subapps. Therefore these mesh divisions are not present in the
+# main app.
+# Explaining results on the sub apps:
+# Each subapp receives results for all its target mesh divisions. They are naturally all the
+# same because they are all matched with the same source app (parent app) source division
+# and the division on the parent app is too small to have more than 1 valid point + value
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  allow_renumbering = false
```

Comment:
Is this necessary?

Reply:
nope. It helps csvdiffs done with samplers, but exodiff is fine

Reply:
resolving because I removed it